Libraries/BatchedBridge/BatchingImplementation/BatchedBridgeFactory.js

```diff
@@ -80,4 +106,13 @@ var BatchedBridgeFactory = {
   }
 };
 
+function _createErrorFromErrorData(errorData: ErrorData): Error {
+  var {
+    message,
+    ...extraErrorInfo,
+  } = errorData;
+  var error = new Error(message);
```

Comment:
Question: do we get a good stack trace when creating the error here? Is there a better place to?